etc/emBright.ini

```diff
@@ -1,22 +1,22 @@
 [Paths]
 coincPath = ./em_bright
-psdPath = ./em_bright
-results = ./em_bright
-logs = ./logs
-numTrials = 5
-wait = 5
+psdPath   = ./em_bright
+results   = ./em_bright
+logs      = ./logs
 
 [EMBright]
-elipsoidSample = 1000
+elipsoidSample   = 1000
 remMassThreshold = 0.0
-fmin = 30
-Forced = False
-chirpmass_cut = 2.612
-lowMass_approx = lalsim.SpinTaylorT4
+
+fmin            = 30
```

Comment:
Note that justifying assignment operators [violates PEP8](https://www.python.org/dev/peps/pep-0008/#pet-peeves).